angular/projects/spark-angular/src/lib/components/inputs/DatePicker.stories.ts

```diff
@@ -79,6 +79,8 @@ export const defaultStory = () => ({
           iconName="calendar"
           additionalClasses="sprk-c-Icon--stroke-current-color"
           sprk-input-icon
+          ariaHidden="true"
+          focusable="false"
```

Comment:
I'm wondering if there's ever a situation where this would need to be different.

Either way, I'm fine with giving people the option and making it a default.

Reply:
I can't think of a scenario where we would want to make the svg focusable (better to wrap it in a button at that point). But I can definitely think of scenarios where they may want the icon to not be hidden and to have an aria-label instead.

Here are some examples I came up with while thinking about this story.

https://codepen.io/bobwise/full/zYNaPVZ